metacat-connector-hive/src/test/groovy/com/netflix/metacat/connector/hive/HiveConnectorUtilSpec.groovy

```diff
@@ -42,4 +45,19 @@ class HiveConnectorUtilSpec extends Specification{
         '1'         | 1L
         '010m'      | 600L
     }
+
+    def "Test for check common view" () {
+        given:
+        def tableInfo = TableInfo.builder().metadata(metadata).build()
+        def isView = HiveTableUtil.isCommonView(tableInfo)
+        def isView2 = HiveTableUtil.isCommonView(tableInfo)
```

Comment:
I think isView2 can be removed.

Reply:
I actually didn't notice this and it's supposed to test a different function. I updated the test and function code. I will merge the code after passing the build.